src/cctag/Detection.cpp

```diff
@@ -876,7 +876,7 @@ void cctagDetection(CCTag::List& markers,
         }
 #endif // WITH_CUDA
 
-        std::vector<cctag::ImageCut> vSelectedCuts[ numTags ];
+        std::vector<std::vector<cctag::ImageCut> > vSelectedCuts( numTags );
```

Comment:
@griwodz pls check if it is a safe change, clang was somehow complaining about making a c-array of vectors. It compiles after this but I hope there are not hidden side effects..